docs/getting-started/welcome.md

```diff
@@ -23,7 +23,7 @@ If you would like to learn more about the concepts behind the DVM and priceless
 
 ## For Developers
 
-If you are a developer interested in building with UMA’s technology, start with this [explainer](synthetic-tokens/explainer.md) of how UMA's priceless synthetic tokens work, and then try out this [tutorial](tutorials/mint-locally.md) for how to deploy your own token on testnet. You can find additional reference materials, including a list of UMA-supported mainnet and testnet deployments, [here](dev-ref/addresses.md).
+If you are a developer interested in building with UMA’s technology, start with this [explainer](synthetic-tokens/explainer.md) of how UMA's priceless synthetic tokens work, and then try out this [tutorial](build-walkthrough/mint-locally) for how to deploy your own token on testnet. You can find additional reference materials, including a list of UMA-supported mainnet and testnet deployments, [here](dev-ref/addresses.md).
 
 ## For Market Makers
```

Comment:
Can we delete this as it isn't one of our primariy users (e.g., token holder, developer)?

Reply:
Sorry, not sure I understand. Delete the whole section? Or edit the 'build-walkthrough' link?